variables.tf

```
@@ -11,6 +11,11 @@ variable "talos" {
       api_server         = string
       kubernetes_version = string
       talos_version      = string
+      extra_mounts       = optional(map(object({
```

```suggestion
      extra_mounts       = optional(list(object({
```
This should be a list instead of a map